.gitignore

```diff
@@ -29,6 +29,22 @@
 playground
 unit_tests
 
+# Dependencies
+catch/
+src/rocksdb-3.10.2.zip
+src/rocksdb/
+src/rocksdb-stamp/
+src/3c8ae031.zip
+src/htslib/
+src/htslib-stamp/
+src/release-0.5.2.zip
+src/yaml-cpp/
+src/yaml-cpp-stamp/
+src/1.*.zip
+
+# Temporary build stuff
+tmp/
+
```

Comment:
These .gitignore changes shouldn't be needed after a recent change in 85611fa46c65 that moved this stuff into an external/ directory. Try deleting it locally or using a fresh checkout.